src/scores.jl

```diff
@@ -86,7 +90,7 @@ function eval_score_on_district(graph::BaseGraph,
     catch e # Check if the user-specified method was constructed incorrectly
         if isa(e, MethodError)
             error_msg = "DistrictScore function must accept graph, array of nodes, and district index."
-            throw(MethodError(error_msg))
+            throw(ArgumentError(error_msg))
```

Comment:
Just a small bugfix during testing - `MethodError`s don't take `msg` as an argument, so I changed it to `ArgumentError`